### ISSUE TITLE
[Features request/suggest] In the spellcheck window 

### ISSUE THREAD

**Issue by Eeel-12** [otsaloma/gaupol#220]
- A "back button" to go back to the previous spellcheck error / just modified error. If the file have lots a Names it's difficult to find 
where i was when i miss click. 
- The Edit button open a new popup window, could be nice if we can edit directly in the current spellcheck window to save some time.
- On spellcheck window open, a "continue where you left of ?" could be nice. I frequently need to exit the spellcheck window to do some search and replace, and because there is lots of names in the file, i have to start spellcheck again from the beginning.

Thank you.

**Comment by otsaloma** [comment]
I agree on these issues, I have noted the same inconveniences in my own use.

* A back button would be useful, but I think a bit difficult to implement on account of the "replace *all*" and "ignore *all*" actions as it's not just a single action that needs to be rolled back, but possibly several.
* The edit button might be removable, that's how it works in the search-replace dialog. It needs to invalidate the various ignore/replace/etc actions as those are not well defined when the text changes. That's probably the reason why I cut corners and did it with a separate dialog as that was easier to get right.
* Continuing where left off would be useful too, but a bit complicated both in terms of both the UI as well as the state management implementation.

I don't expect to work on these but am open to contributions.

**Comment by Eeel-12** [comment]
> * A back button would be useful, but I think a bit difficult to implement on account of the "replace _all_" and "ignore _all_" actions as it's not just a single action that needs to be rolled back, but possibly several.

Thank you for your feedback.
maybe a back button withtout undo ? I mean just go back to previous modified entry to get the oportunity to edit it again.

I don't have codding knowledge, maybe if all requests are visible on landing page, someone with enough skills would be interested in improving.

**Comment by otsaloma** [comment]
> The Edit button open a new popup window, could be nice if we can edit directly in the current spellcheck window to save some time.

Done in 40212fffee24ca984ced1bab89429cc632d450b2
